tests/NHSD.BuyingCatalogue.SolutionLists.Persistence.DatabaseTests/SolutionListRepositoryTests.cs

```diff
@@ -193,47 +202,50 @@ await SolutionCapabilityEntityBuilder.Create()
         [Test]
         public async Task ShouldListMultipleSolutionsWithCapabilities()
         {
-            await SolutionEntityBuilder.Create()
+            await CatalogueItemEntityBuilder
+                .Create()
+                .WithCatalogueItemId(Solution1Id)
                 .WithName("Solution1")
-                .WithId(Solution1Id)
                 .WithSupplierId(_supplierId)
                 .WithPublishedStatusId(3)
                 .Build()
                 .InsertAsync();
 
             await SolutionEntityBuilder.Create()
+                .WithId(Solution1Id)
+                .WithSummary("Sln1Summary")
+                .Build()
+                .InsertAsync();
+
+            await CatalogueItemEntityBuilder
+                .Create()
+                .WithCatalogueItemId("Sln2")
                 .WithName("Solution2")
-                .WithId("Sln2")
                 .WithSupplierId(_supplierId)
                 .WithPublishedStatusId(3)
                 .Build()
                 .InsertAsync();
 
             await SolutionEntityBuilder.Create()
-                .WithName("Solution3")
-                .WithId("Sln3")
-                .WithSupplierId(_supplierId)
-                .WithPublishedStatusId(1)
+                .WithId("Sln2")
+                .WithSummary("Sln2Summary")
                 .Build()
                 .InsertAsync();
 
-            await SolutionDetailEntityBuilder.Create()
-                .WithSolutionId(Solution1Id)
-                .WithSummary("Sln1Summary")
-                .Build()
-                .InsertAndSetCurrentForSolutionAsync();
-
-            await SolutionDetailEntityBuilder.Create()
-                .WithSolutionId("Sln2")
-                .WithSummary("Sln2Summary")
+            await CatalogueItemEntityBuilder
+                .Create()
+                .WithCatalogueItemId("Sln3")
+                .WithName("Solution3")
+                .WithSupplierId(_supplierId)
+                .WithPublishedStatusId(3)
```

Comment:
Potential change for review... Just noticed that the previous file used `.WithPublishedStatusId(1)`, this looks like it has been changed to status `.WithPublishedStatusId(3)`.

Reply:
The publication status isn't tested as part of this test so the value used won't affect the outcome in this case. As such, it might be better just to omit the line.